client/reader/site-stream/featured.jsx

```diff
@@ -63,14 +68,14 @@ class FeedFeatured extends React.PureComponent {
 					};
 
 					return (
-						<div
+						<button
```

Comment:
maybe make this a big anchor instead of a button? It has a URL after all.

Reply:
That's actually a better idea than my `button`, but also scuppers all the styles, especially in the small viewport version 🤦‍♂️

I'll ship this for now and bear it in mind for future improvements 👍